.github/workflows/meson.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-13]
         arch: ['x64']
-        version: ['13']
+        version: ['11']
```

Comment:
Can we keep GCC 13 please

Reply:
You need a Julia version compiled with GCC 13 and it's not yet the case on Windows. Is it fine if we use GCC13 except Windows where we keep GCC 11?

Reply:
Ah okay yeah sure that's fine!

Reply:
I will change in another PR. Csn I merge Jari?

Reply:
Sure sure